zarpaint/napari.yaml

```diff
@@ -0,0 +1,39 @@
+name: zarpaint
+schema_version: 0.1.0
+contributions:
+  commands:
+  - id: zarpaint.add_points_3d_with_alt_click
+    title: Create add_points_3d_with_alt_click
```

Comment:
```suggestion
    title: Add points with alt-click in 3D
```

Reply:
Wait, I just noticed that these are commands, not widgets. Can npe2 already add parameter-less function calls without a widget? If so, can you convert this to just a command that runs without the widget?

Reply:
Sadly, no, you still need the widget. The `command` contribution is associated with the `widget` contribution via the `command_id`.